src/test/java/com/SAS/acceptanceTests/LeagueManagementAT.java

```diff
@@ -1,81 +1,207 @@
+        System.out.println("System saved the data");
+        Assertions.assertTrue(league.getRankPolicy(season).getName().equals(rPolicy.getName()));
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.5.a test - success
+    @Test
+    void successfulDefineGamesPolicyForTheLeague() {
+        //9.6
+        long startTime = System.nanoTime();
+        successfulDefineSeasonToLeague();
+        System.out.println("Choose games policy");
+        league.addGamePolicy(season, gPolicy);
+        System.out.println("You have chose: " + gPolicy.toString());
+        System.out.println("System saved the data");
+        Assertions.assertTrue(league.getGamesPolicy(season).getName().equals(gPolicy.getName()));
+        System.out.println("League initialized successfully");
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.1.b test - failure
+    @Test
+    void failingLeagueCreation() {
+        //9.1
+        long startTime = System.nanoTime();
+        System.out.println("You have selected to edit association settings");
+        Assertions.assertFalse(false);
+        System.out.println("You are not authorized to enter this page");
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.2.b test - failure
+    @Test
+    void failingDefineSeasonToLeague() {
+        //9.2
+        long startTime = System.nanoTime();
+        System.out.println("You have selected to edit association settings");
+        System.out.println("There is the association page");
+        Assertions.assertFalse(CRUD.isLeagueExist(new League("Liga Leumit")));
+        System.out.println("The league: \"Liga Leumit\" was not found");
+        long endTime = System.nanoTime();
+        System.out.println("The time the test took is:" + ((double) (endTime - startTime) / 1000000) + " ms");
+    }
+
+    //9.4.b test - failure
+    @Test
+    void assignRefereeToLeagueInSeason() {
+        //9.4
+        long startTime = System.nanoTime();
+        System.out.println("You have selected to edit association settings");
+        System.out.println("This is the association page");
+        System.out.println("Pressed \"Referee management\"");
+        System.out.println("This is the referees page");
+        System.out.println("You chose to add referee");
+        System.out.println("Please choose the league and seas you on the referee to be added");
+        league.addSeason(season);
+        league.addReferee(season, referee);
+        if (league.getReferees().get(season).contains(referee)) {
+            System.out.println("The referee is already assigned for the league in this season");
```

Comment:
I don't understand why this test fails, I think you need to define this referee for the season in the beginning of the test under setUp comment. If this is suppose to be a referee that already exists - I would expect the function of add referee to season would return false - and than check it with asserFalse().